### PR TITLE
Mov 8/add ability to use blur for holders

### PR DESCRIPTION
**Pull request by StasAntonov**

### CODE REVIEW

File: app/src/main/res/drawable/ic_test_back.jpg
Comment (IhorPashkovskyi) [inline app/src/main/res/drawable/ic_test_back.jpg:1]: remove this image

File: app/src/main/res/drawable/ic_test_back_2.jpg
Comment (IhorPashkovskyi) [inline app/src/main/res/drawable/ic_test_back_2.jpg:1]: remove this image

File: app/src/main/res/drawable/ic_test_back.jpg
Comment (StasAntonov) [reply app/src/main/res/drawable/ic_test_back.jpg:1]: removed

File: app/src/main/res/drawable/ic_test_back_2.jpg
Comment (StasAntonov) [reply app/src/main/res/drawable/ic_test_back_2.jpg:1]: removed

**Pull request merged**